build/configs/artik05x/artik05x_download.sh

```diff
@@ -145,11 +145,18 @@ compute_ocd_commands()
 
 download()
 {
-        parts_default=`grep -A 2 'config FLASH_PART_NAME' ${PARTITION_KCONFIG} | sed -n 's/\tdefault "\(.*\)".*/\1/p'`
-        parts2=${CONFIG_FLASH_PART_NAME:=${parts_default}}
-        parts=`echo $parts2 | sed "s/,/ /g"`
+	parts=$1;
 
-        # Make Openocd commands for parts
+	if [[ -n $parts ]] && [[ "$parts" != "all" ]] && [[ "$parts" != "ALL" ]]
+	then
+		echo "##Download $parts"
+	else
+	        parts_default=`grep -A 2 'config FLASH_PART_NAME' ${PARTITION_KCONFIG} | sed -n 's/\tdefault "\(.*\)".*/\1/p'`
+		parts2=${CONFIG_FLASH_PART_NAME:=${parts_default}}
```

Comment:
To avoid confusing of words `parts_default` and `part2`, how about using below?
`default_parts` and `configured_parts`

Reply:
Okay